lib/helper-functions.js

```diff
@@ -20,3 +22,14 @@ const componentNameToMacroName = componentName => {
 }
 
 exports.componentNameToMacroName = componentNameToMacroName
+
+// Converts directory to object
+const directoryToObject = (path) => {
```

Comment:
This file's turning into a bit of a dumping ground (or at least it will have, once the test changes are merged in as well) – can we think about splitting it up or refactoring?

Reply:
I agree, we can split helper functions in different files inside `lib`.